README.md

```diff
@@ -843,7 +843,7 @@ render status: :forbidden
     User.find_by_first_name_and_last_name('Bruce', 'Wayne')
```

Comment:
Do you think it makes sense to update the desction to "Favor the use of `find_by`/`find`"?

Reply:
It makes sense, but the below suggestion seems be more careful 🤠.